### PR TITLE
MIG 1040 Document additional excluded resources

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
MIG 1040 Document additional excluded resources

https://issues.redhat.com/browse/MIG-1040

MTC 1.7

Preview: https://deploy-preview-43248--osdocs.netlify.app/openshift-enterprise/latest/migration_toolkit_for_containers/advanced-migration-options-mtc.html#migration-excluding-resources_advanced-migration-options-mtc

### CODE REVIEW

**Review by jmontleon — APPROVED** [review]

**Review by stillalearner — APPROVED** [review]
QE approved.

File: modules/migration-excluding-resources.adoc [hunk 1]
```diff
@@ -32,25 +32,15 @@ metadata:
 spec:
   disable_image_migration: true <1>
   disable_pv_migration: true <2>
+  additional_excluded_resources: <3>
+  - foo
```
Comment (nalhadef) [inline modules/migration-excluding-resources.adoc:36]: Don't use "fubar" or variations of it.

image

https://www.ibm.com/docs/en/ibm-style?topic=word-usage#f

File: modules/migration-excluding-resources.adoc [hunk 2]
```diff
@@ -68,5 +58,5 @@ The output contains the excluded resources:
 ----
     - name: EXCLUDED_RESOURCES
       value:
-      imagetags,templateinstances,clusterserviceversions,packagemanifests,subscriptions,servicebrokers,servicebindings,serviceclasses,serviceinstances,serviceplans,imagestreams,persistentvolumes,persistentvolumeclaims
+      foo,bar,imagetags,templateinstances,clusterserviceversions,packagemanifests,subscriptions,servicebrokers,servicebindings,serviceclasses,serviceinstances,serviceplans,imagestreams,persistentvolumes,persistentvolumeclaims
```
Comment (nalhadef) [inline modules/migration-excluding-resources.adoc:61]: See previous fubar comment.

Is the list of values be indented like the lines above it?

**Review by nalhadef — COMMENTED** [review]
Remember to squash your commits.

File: modules/migration-excluding-resources.adoc (hunk 2 above)
Comment (sbeskin-redhat) [reply modules/migration-excluding-resources.adoc]: @nalhadef  
Corrections made, commits squashed. Thank you!

**Review by stillalearner — COMMENTED** [review]
/lgtm

**Pull request merged**